<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAnnotationWidgetBase</class>
 <widget class="QWidget" name="QgsAnnotationWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QCheckBox" name="mMapPositionFixedCheckBox">
     <property name="text">
      <string>Fixed map position</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mMapMarkerLabel">
       <property name="text">
        <string>Map marker</string>
       </property>
       <property name="buddy">
        <cstring>mMapMarkerButton</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mMapMarkerButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mFrameWidthLabel">
       <property name="text">
        <string>Frame width</string>
       </property>
       <property name="buddy">
        <cstring>mFrameWidthSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="mFrameWidthSpinBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mBackgroundColorLabel">
       <property name="text">
        <string>Background color</string>
       </property>
       <property name="buddy">
        <cstring>mBackgroundColorButton</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsColorButton" name="mBackgroundColorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mFrameColorLabel">
       <property name="text">
        <string>Frame color</string>
       </property>
       <property name="buddy">
        <cstring>mFrameColorButton</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsColorButton" name="mFrameColorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
